test/byContract/IexecAccessors/IexecAccessors.test.ts

```diff
@@ -110,6 +194,25 @@ describe('IexecAccessors', async () => {
         expect(task.resultsCallback).to.equal('0x');
     });
 
+    it('viewContribution', async function () {
+        const { dealId, taskIndex, taskId } = await createDeal();
+        await iexecWrapper.initializeTask(dealId, taskIndex);
+        await iexecWrapper.contributeToTask(dealId, taskIndex, resultDigest, worker1);
+        const contribution = await iexecPoco.viewContribution(taskId, worker1.address);
+        expect(contribution.status).to.equal(ContributionStatusEnum.CONTRIBUTED);
+        expect(contribution.resultHash.length).to.equal(66);
+        expect(contribution.resultSeal.length).to.equal(66);
```

Comment:
To understand, why checking length instead of content?

Reply:
I wanted to reduce noise in the test and not having to worry about the hashes. I assert that a hash is returned without necessarily checking its content as other tests (contribution tests) are already doing that.